src/server/editorServices.ts

```diff
@@ -1341,20 +1348,19 @@ namespace ts.server {
                 const parsedCommandLine = ts.parseJsonConfigFileContent(rawConfig.config, this.host, dirPath, /*existingOptions*/ {}, configFilename);
                 Debug.assert(!!parsedCommandLine.fileNames);
 
-                if (parsedCommandLine.errors && (parsedCommandLine.errors.length > 0)) {
-                    return { succeeded: false, errors: parsedCommandLine.errors };
-                }
-                else if (parsedCommandLine.fileNames.length === 0) {
+                if (parsedCommandLine.fileNames.length === 0) {
                     const error = createCompilerDiagnostic(Diagnostics.The_config_file_0_found_doesn_t_contain_any_source_files, configFilename);
-                    return { succeeded: false, errors: [error] };
+                    return { succeeded: false, errors: concatenate(parsedCommandLine.errors, [error]) };

```

Comment:
even if it is invalid, why should not we build one? we found  tsconfig.json, and we know the open file. so we can create a project with just that..